src/azure-cli/azure/cli/command_modules/backup/custom_base.py

```diff
@@ -322,10 +376,16 @@ def disable_auto_for_azure_wl(client, resource_group_name, vault_name, item_name
 
 def restore_disks(cmd, client, resource_group_name, vault_name, container_name, item_name, rp_name, storage_account,
                   target_resource_group=None, restore_to_staging_storage_account=None, restore_only_osdisk=None,
-                  diskslist=None, restore_as_unmanaged_disks=None, use_secondary_region=None):
+                  diskslist=None, restore_as_unmanaged_disks=None, use_secondary_region=None, rehydration_duration=15,
+                  rehydration_priority=None):
+
+    if rehydration_duration < 10 or rehydration_duration > 30:
+        raise InvalidArgumentValueError('--rehydration-duration must have a value between 10 and 30.')
```

Comment:
"value between" may create confusion that the extremes are included or not.

Reply:
Have added both inclusive in brackets: --rehydration-duration must have a value between 10 and 30 (both inclusive).

Now it's clear i think.